bindgen/clang.rs

```diff
@@ -973,6 +974,20 @@ impl Cursor {
     pub(crate) fn is_inline_namespace(&self) -> bool {
         unsafe { clang_Cursor_isInlineNamespace(self.x) != 0 }
     }
+
+    /// Obtain the number base (radix) of a literal definition corresponding to the cursor.
+    ///
+    /// Returns `None` if unable to infer a base.
+    pub(crate) fn get_literal_radix(&self) -> Option<LiteralRadix> {
+        self.cexpr_tokens()
+            .iter()
+            .filter(|cexpr_token| {
+                cexpr_token.kind == cexpr::token::Kind::Literal
+            })
+            .find_map(|lit_tok| {
+                LiteralRadix::from_literal_token_raw(&lit_tok.raw)
+            })
+    }
```

Comment:
Just found a bug, most likely caused by this. When multiple integers are declared in the same statement, all of them incorrectly get the radix of the first one. I'll rework the solution and add some testing.